frontend/src/app/components/notifications/notifications.component.ts

```diff
@@ -61,7 +63,11 @@ export class NotificationsComponent implements OnInit {
     }
 
     if (this.isValid(notification)) {
-      this.notificationsService.pushNotification(notification).subscribe(x=>console.log(x));
+      const message = this.translate.instant("NOTIFICATION_SEND_CONFIRMATION");
+      if (!confirm(message.replace('<!nr!>',this.selectedObserversIds.length))) {
```

Comment:
Why not use something tried and tested before like `%d` instead of `<!nr!>`? I'm thinking that it'll be more familiar to contributors.